include/libtorrent/aux_/session_impl.hpp

```diff
@@ -180,7 +176,10 @@ namespace libtorrent
 		struct tracker_logger;
 #endif
 
-		TORRENT_EXPORT std::pair<bencode_map_entry*, int> settings_map();
+#ifndef TORRENT_DISABLE_DHT
+		TORRENT_EXTRA_EXPORT dht_settings read_dht_settings(bdecode_node const& e);
```

Comment:
oh, I see ``dht_settings`` doesn't seem to live in the ``dht`` namespace either..